nemo_curator/classifiers/aegis.py

```diff
@@ -199,7 +199,7 @@ def __init__(
         self.keep_raw_pred = keep_raw_pred
 
         try:
-            model = AegisHFModel(config=config)
+            model = AegisHFModel(config=config, max_mem_gb=48)
```

Comment:
Any particular reason you are setting this again here? If not, I'd prefer to keep it at default. If we need to change it later, I can see finding this hardcoded value a bit tricky potentially.

Reply:
Did not want to mess with default. Ideally i would want `Aegis` to fall back to `max memory_db` default too (which you can control) via a flag than hard coding it , what do you think ?

Reply:
Waiting on your followup here. Have resolved everything else.

Reply:
Oh if you mean introducing a variable in the constructor for `AegisClassifier` like the other classifiers, then yeah I agree we should do that.

Reply:
Resolved by : https://github.com/NVIDIA/NeMo-Curator/commit/18d93da0d6193796339d1b89b457c88766d64a7d